console/src/main/java/org/georchestra/console/dto/EventType.java

```diff
@@ -2,6 +2,8 @@
 
 /**
  * Type discriminator for application events
+ * <p>
+ * TODO: dead code!
```

Comment:
grepping the code, it does not seem to be used somewhere indeed, I think we can remove the class.